typed-racket-lib/typed-racket/base-env/prims-contract.rkt

```diff
@@ -391,9 +393,13 @@
            #,(if (attribute ne)
                  (internal (syntax/loc stx (define-type-alias-internal ty (Opaque pred))))
                  (syntax/loc stx (define-type-alias ty (Opaque pred))))
-           #,(ignore #'(define pred-cnt
-                         (or/c struct-predicate-procedure?/c
-                               (any-wrap-warning/c . c-> . boolean?))))
+           #,(if (attribute unsafe)
+                 (ignore #'(define pred-cnt
```

Comment:
No, really, let's not use a contract. For example, when testing the performance impact of contracts, it's nice to be able to use `unsafe-require/typed`. Sometimes adding contracts defeats that.